lib/ide/parser/src/api.rs

```diff
@@ -8,9 +9,11 @@ pub type Ast = ast::Ast;
 // == Parser ==
 // ============
 
+pub type IDMap = Vec<((isize, isize), ID)>;
+
 /// Entity being able to parse Luna programs into Luna's AST.
 pub trait IsParser {
-    fn parse(&mut self, program: String) -> Result<Ast>;
+    fn parse(&mut self, program: String, ids: IDMap) -> Result<Ast>;
```

Comment:
The original code wasn't complaint in the first place (it was moved to this repo without refactoring) but when you touch it, please follow the coding guidelines.

There should be no space between the parameter name and its type (so `ids:IDMap` and so on).

Reply:
Not yet resolved.